docs/ops/pooling/MaxPool_1.md

```diff
@@ -6,7 +6,79 @@
 
 **Short description**: [Reference](http://caffe.berkeleyvision.org/tutorial/layers/pooling.html)
 
-**Detailed description**: [Reference](http://cs231n.github.io/convolutional-networks/#pool)
+**Detailed description**: [Reference](https://deeplizard.com/learn/video/ZjM_XQa5s6s). Input shape can be either 3D, 4D or 5D. Pooling operation is performed with the respect to input shape from the third dimension to the last dimension. If paddings are used then during the pooling calculation their value is `-inf`.  
```

Comment:
1. Please make the description first and a link at last.
2. Why you list different links in the detailed description and short description? Please describe where does this link go.
3. Also, please mention max pooling instead of just pooling.
4. Earlier we don't have such limitation in the specification: 3D, 4D, or 5D.

Reply:
Link in short description goes to the short description of the operator and link in the detailed description goes to article. 
While I was working on average pool I was told to change the limitation to 3D, 4D and 5D and then do the same for max pooling. I made those changes in different PR, which is in review right now.

Reply:
After adding the additional information about how the operator work, I think that the link in the short description is not needed anymore.